src/main/java/SkyPro/ExaminerApp/service/ExaminerServiceImpl.java

```diff
@@ -1,23 +1,23 @@
 package SkyPro.ExaminerApp.service;
 
+import SkyPro.ExaminerApp.Domain.Question;
 import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
 import org.springframework.web.server.ResponseStatusException;
-import skypro.examinerservice.domain.Question;
+
 
 import java.util.*;
 
 @Service
 public class ExaminerServiceImpl implements ExaminerService {
 
     private final QuestionService questionService;
-
     public ExaminerServiceImpl(QuestionService questionService) {
         this.questionService = questionService;
     }
 
     @Override
-    public Collection<Question> getQuestions(int amount) {
+    public Collection getQuestions(int amount) {
```

Comment:
надо указывать дженерик-тип
```suggestion
    public Collection<Question> getQuestions(int amount) {
```